openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -144,6 +162,10 @@ def process(self, instance):
 
             path = capture.capture(log=self.log, **preset)
 
+        # Restoring viewport options.
+        for key, value in viewport_defaults.items():
+            cmds.modelEditor(panel, edit=True, **{key: value})
```

Comment:
```suggestion
        cmds.modelEditor(panel, edit=True, **viewport_defaults)
```
maybe this will also work?

Reply:
Nice one!